detections/endpoint/windows_suspicious_process_file_path.yml

```diff
@@ -1,7 +1,7 @@
-name: Suspicious Process File Path
+name: Windows Suspicious Process File Path
```

Comment:
This rename needs a deprecation/new detection, otherwise you're deleting the old detection and supplying a new one all at once.

Reply:
thanks @ljstella .I already fixed this issue.